main.go

```diff
@@ -55,23 +55,23 @@ func main() {
 		"listen", listen,
 		"oauth.provider", cfg.GenOAuth.Provider)
 
-	mux := mux.NewRouter()
+	theMux := mux.NewRouter()
```

Comment:
why `theMux`?

Reply:
My IDE  was telling me those were names overlapping with some builtin stuff. So it was the simplest way to change them.  They can be anything but what they were.